src/coreclr/src/jit/lsra.cpp

```diff
@@ -1563,6 +1562,7 @@ void LinearScan::identifyCandidates()
         return;
     }
 
+    VarSetOps::AssignNoCopy(compiler, exceptVars, VarSetOps::MakeEmpty(compiler));
```

Comment:
Why was that line moved from `identifyCandidatesExceptionDataflow` to its caller `identifyCandidates`?
The only difference that I see is that we now have it empty when `(compiler->compHndBBtabCount <= 0)` and we don't visit `identifyCandidatesExceptionDataflow`, is that the case?
However, in this case I do not see any uses of it, they are all protected with the same condition `compiler->compHndBBtabCount > 0`.

Reply:
In my EHWriteThru changes I have modified some uses to make it both cleaner and a bit more efficient. Also, leaving it uninitialized results in weird failures when one forgets to check.